cl_dll/in_camera.cpp

```diff
@@ -1,4 +1,4 @@
-//========= Copyright � 1996-2002, Valve LLC, All rights reserved. ============
+//========= Copyright © 1996-2002, Valve LLC, All rights reserved. ============
```

Comment:
Cool, just remove this diff whatever it is and it's good

Reply:
After an epic git add -p war and force push, this diff disappeared in the overall PR. Gotta figure out the encoding stuff in my VSCode, sorry. :|